src/framework/theme/component/application/applicationProvider.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react';
+import {
+  ThemeMappingType,
+  ThemeMapType,
+} from 'eva/packages/common';
+import { MappingProviderProps } from '../mapping';
+import { ThemeProviderProps } from '../theme';
+import { ThemeType } from '../../type';
+import { ModalPanel } from '../modal';
+import { StyleProvider } from '../style';
+
+interface ApplicationProviderProps {
+  children: JSX.Element | ReactNode | null;
+}
+
+export type Props = MappingProviderProps & ThemeProviderProps & ApplicationProviderProps;
+
+interface State {
+  mapping: ThemeMappingType;
+  styles: ThemeMapType;
+  theme: ThemeType;
+}
+
+export class ApplicationProvider extends React.Component<Props, State> {
+
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      mapping: props.mapping,
+      styles: props.styles,
+      theme: props.theme,
+    };
+  }
+
+  render() {
+   return (
+     <ModalPanel>
```

Comment:
Can this modal be styled?

Reply:
I guess this component will be the separate UI component in the future